src/pages/documentation/adobe-journey-optimizer/release-notes.md

```diff
@@ -1,10 +1,16 @@
 # Release Notes
 
+## February 3, 2023
+
+### Android Messaging 2.0.0
+
+* Initial release to support [Adobe Journey Optimizer](./index.md) for Adobe Experience Platform Mobile SDKs for Android compatible with Mobile Core 2.0.0. This library is available as an [open sourced project on GitHub](https://github.com/adobe/aepsdk-messaging-android).
```

Comment:
Not an initial release for Messaging on Android as well.

Reply:
hmm.. I was referring to initial release compatible with Mobile Core 2.0.0, let me know if that was not clear and if I should rephrase?

Reply:
I think it would be better to call it as a major version update to support Mobile Core 2.0.0, something like - "Major version update for [Adobe Journey Optimizer](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-messaging-android)."

Reply:
Do you mean call it Major version update for all extensions then? 
And for the link:
If the extension was open source before use: `This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-messaging-android).`
If the extension is newly open sourced use: `This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-edge-android)."`

Reply:
Yeah I've been debating that myself and thinking out loud here. Do not want to call it an initial release for already open sourced extensions atleast. What do you think about calling it a major version update to support Mobile Core 2.0.0 for all? For the link, I'm ok if we want to make a distinction or just omit it for projects that are already open sourced.

Reply:
Major version update for all sounds good to me. And as a reminder I will keep the link even for already open sourced projects with the difference in wording as per my previous comment. Will update all soon

Reply:
Sounds good. Thanks @emdobrin. We need to add additional info for a few extensions where some changes are not backwards compatible - both in the release notes and FAQs. I will update those once your pr is merged.